mysql-test/suite/binlog/t/binlog_mysqlbinlog_warn_stop_datetime.test

```diff
@@ -23,10 +23,6 @@ flush logs;
 --echo # MYSQL_BINLOG --short-form --stop-datetime='2001-09-08 21:46:50' MYSQLD_DATADIR/master-bin.000001 --result-file=ignored_output_file
 --exec $MYSQL_BINLOG --short-form --stop-datetime='2001-09-08 21:46:50' $MYSQLD_DATADIR/master-bin.000001 --result-file=$ignored_output_file 2>&1
 
---echo Case: Stop datetime between records, must not see warning.
```

Comment:
This only looks like a duplicate superficially, but the behavior is different because there's a binlog entry now from the prior case.  Please restore this case.

Reply:
I retract that, it is correct to remove this case, thanks!